### PR TITLE
Fix NewSpecGenerator args in pkg/bindings/test

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@baude @vrothberg @giuseppe PTAL

### CODE REVIEW

File: pkg/bindings/test/common_test.go [hunk 1]
```diff
@@ -3,13 +3,14 @@ package test_bindings
...
 	"fmt"
-	"github.com/containers/libpod/libpod/define"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
 
+	"github.com/containers/libpod/libpod/define"
+
```
[the remark this answers is not in the document]
Comment (lsm5) [reply pkg/bindings/test/common_test.go]: done, thanks

**Comment by mheon** [comment]
How did CI not catch this? Regardless, LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
I wonder too @mheon

**Comment by lsm5** [comment]
Many tests in pkg/bindings/containers_test.go are failing and this was the first issue I noticed. So, wonder if these tests are even being run..

**Pull request merged**